cmd/operator-ui/templates/blackducks/index.html

```diff
@@ -32,47 +32,53 @@ <h4>Black Duck Instances</h4>
     </thead>
     <tbody>
       <%= if (len(blackducks) > 0) { %>
-        <%= for (blackduck) in blackducks { %>
-          <tr>
-            <td><%= blackduck.Name %></td>
-            <td><%= blackduck.Spec.Namespace %></td>
-            <td><%= blackduck.Spec.Size %></td>
-            <td><%= blackduck.Spec.Version %></td>
-            <td><%= blackduck.Spec.Type %></td>
-            <td><%= blackduck.Spec.DbPrototype %></td>
-            <%= if (blackduck.Spec.PersistentStorage) { %>
-              <td><%= blackduck.Spec.PersistentStorage %>-<%= blackduck.Spec.PVCStorageClass %>-<%= blackduck.Spec.ScanType %></td>
+      <%= for (blackduck) in blackducks { %>
```

Comment:
@yashbhutwala  why the format got changed?

Reply:
That's what `VS Code` formatted it to :)